sources/packages/forms/src/form-definitions/exportfinancialreports.json

```diff
@@ -115,6 +110,10 @@
                       {
                         "label": "Forecast disbursements",
                         "value": "Disbursement_Forecast_Report"
+                      },
+                      {
+                        "label": "Disbursements",
+                        "value": "Disbursement_Report"
```

Comment:
I know the `"Disbursement_Forecast_Report"` is always not following camelCase.  I believe we decided to use camelCase in both camunda and forms too. but may I know the reason, not to use camelCase?

Reply:
The reason it would have happened is, formio auto creats the id when the value is created in the data tab. We can override it, so I followed it as it was before. Lets discuss and change. @andrewsignori-aot @henryoforeh-aot @dheepak-aot what you guys feel to change the id here?

Reply:
This one is targeting a DB column "unique identifier" (which could also be defined as camelCase). Maybe because it is not targeting a variable name we can keep it as it is. I believe that it is also used to name the downloaded file.
That said, I missed that in the previous PR and I am not sure if there is a strong reason to have this format, even using the "_" that is not so common in the solution while naming whatever. Maybe the file name is the only possible explanation.

Reply:
Yes it is @andrewsignori-aot . The file name is derived from this value dynamically.